tests/test-pool-search.rb

```diff
@@ -165,7 +165,46 @@
   #
   # test that a date range includes an expected item
   #
-  #
+  it "#{url} should return one or more items" do
```

Comment:
"should return one or more items" isn't terribly helpful as a message.  Perhaps "{url} should support searching by date range"